atest/robot/running/for.robot

```diff
@@ -289,6 +361,11 @@ Should Be For In Range Keyword
     Should Be For Keyword    ${kw}    ${subcount}
     Should Contain    ${kw.name}    IN RANGE    Not FOR IN RANGE keyword
 
+Should Be For In Zip Keyword
+    [Arguments]    ${kw}    ${subcount}
+    Should Be For Keyword    ${kw}    ${subcount}
+    Should Contain    ${kw.name}    IN ZIP    Not FOR IN RANGE keyword
```

Comment:
The configured error message is wrong. It's not that useful here but definitely should be correct if it is used.